docs/implementation/rhcc.md

```diff
@@ -0,0 +1,24 @@
+# Red Hat Container-First Content
+
+
+|            |          |
+|:-----------|:---------|
+|Indexer Type|`package` |
```

Comment:
I've kind of been going down the `Detector` route but I don't know if that's making things more confusing now